policy-engine/src/main.rs

```diff
@@ -103,6 +104,7 @@ fn main() -> Result<(), Error> {
                     .route(actix_web::web::get().to(openapi::index)),
             )
     })
+    .keep_alive(10)
```

Comment:
What does this do?

Reply:
Same but for policy-engine: if the client is hitting PE with requests it should reuse connections instead of closing / creating new ones. The latter is an expensive operation and might help PE to cope with malicious client (== load testing scripts)

Reply:
I trust you to have included this in the commit message ;-)

Reply:
> ... if the client is hitting PE with requests it should reuse connections instead of closing / creating new ones.

But we expect the cluster-version operator to poll the policy engine every 2 to 4 minutes.  Do we expect the CVO to hold those open for that long?  Creating new connections doesn't seem like it would be _that_ expensive :p.

Reply:
On PE's side this measure is against malicious clients, which create a lot of connections and close them instantly.

CVO requests probably won't be affected by this - our average payload is ~15KB, so it should be almost instant on any connection.